howtos/introduction-of-a-new-tc-member.md

```diff
@@ -0,0 +1,12 @@
+# Introduction of a new TC member
+
+Checklist for introducing a new Technical Committee member
+
+- Become a owner of Eiffel Community GitHub organization
+- Add email as billing email under [billing settings](https://github.com/organizations/eiffel-community/settings/billing)
+- Become a maintainer of eiffel-community-maintainers in the [maintainers list](https://github.com/orgs/eiffel-community/teams/eiffel-community-maintainers/members)
+- Become a owner of Eiffel Community Google Groups in the [members list](https://groups.google.com/g/eiffel-community/members)
+- Become a owner of Eiffel Technical Committee Google Groups in the [members list](https://groups.google.com/g/eiffel-tc/members)
+- Become a member of Eiffel Community HackMD team in the [members list](https://hackmd.io/team/eiffel-community/manage#members)
+- Edit access to the YouTube channel

```

Comment:
This is slightly ambiguous since there's an Editor role but we're currently Managers. Suggestion:

> Be granted the Manager role of the YouTube channel

Reply:
Hmm. Or should it be _in_ the YT channel?

Reply:
Reworded in 47fd43f